bundles/EcommerceFrameworkBundle/Model/AbstractVoucherSeries.php

```diff
@@ -26,22 +26,20 @@ abstract class AbstractVoucherSeries extends \Pimcore\Model\DataObject\Concrete
     abstract public function getTokenSettings();
 
     /**
-     * @return bool|TokenManagerInterface
-     *
-     * @throws \Pimcore\Bundle\EcommerceFrameworkBundle\Exception\InvalidConfigException
+     * @return TokenManagerInterface|null
      */
     public function getTokenManager()
     {
         $items = $this->getTokenSettings();
-        if ($items && $items->get(0)) {
 
+        if ($items->get(0)) {
             // name of fieldcollection class
             $configuration = $items->get(0);
 
             return Factory::getInstance()->getTokenManager($configuration);
         }
 
-        return false;
+        return null;
```

Comment:
if we change this, it also needs to be considered [here](https://github.com/pimcore/pimcore/blob/254d1d7830bac4d6fddc244d931c2f769f4a3c74/bundles/EcommerceFrameworkBundle/VoucherService/DefaultService.php#L328)

Reply:
add it here: https://github.com/pimcore/pimcore/pull/9067/commits/054e8f6cbd1b3dca2c1103c0e7e6a13f3bd0a53b